### ISSUE TITLE
Is the Smartsheet gem abandoned?

### ISSUE THREAD

**Issue by brandondrew** [smartsheet-platform/smartsheet-ruby-sdk#59]
The Smartsheet gem has not seen any updates in about a year.  This is not necessarily automatically a problem, but considering that it is pegged to an ancient version of Faraday, despite issues filed, this leads one to guess that it has been abandoned.  

This sort of can create all sorts of headaches, 

```
Bundler could not find compatible versions for gem "faraday":
  In Gemfile:
    faraday-http was resolved to 1.0.1, which depends on
      faraday (~> 1.0)

    smartsheet (>= 1.3.0) was resolved to 2.86.1, which depends on
      faraday (~> 0.13.1)
```

but I mention this in passing, as the point of _this_ issue is just to inquire on the status. 

Could you post a status update?  Can we expect issues to be dealt with, or are all maintainers currently swamped with other responsibilities?  Do we need to fork it?  Will you accept pull requests? 

Perhaps some of us can help, but it would be awesome if we knew where to direct our energies.  

Thanks!

**Comment by armstnp** [comment]
I wish I knew, as well. My guess is that this gem is used little enough (based on the stargazers and Rubygems stats) compared to the other language bindings that it is no longer a priority for support.

**Comment by rahearn** [comment]
Well, there's "not a priority" levels of support, and "let #58 sit for 5 months with no response" levels of support. This is abandoned until proven otherwise.

**Comment by brandondrew** [comment]
@armstnp  @rahearn  if I forked it, could I count on any contributions from you?

**Comment by rahearn** [comment]
Unfortunately, I cannot commit to that. The project I was using the gem on was just retired and I don't have time to maintain anything that I'm not using day-to-day.

**Comment by armstnp** [comment]
I don't believe I will either. This gem was originally created based on a combination of internal and external documentation about the API, and without that at the present, it's unlikely new features would be as readily adaptable. Combined with the low adoption rate of the gem, I would probably not plan on contributing further.

**Comment by brandonzylstra** [comment]
THIS IS SO HORRIBLY ABANDONED...
